feat: init compendium (#42)
atomone-hub/genesis: COMPENDIUM.md

| [<ins>#21: ATOM-ONE Secure Communications for Users</ins>](https://github.com/atomone-hub/genesis/issues/21)         | COverS8              | Need for secure communication alternatives to Telegram and Discord. Proposal for a secure app for official governance communications.               |
| [<ins>#22: Link with Tendermint Atom One</ins>](https://github.com/atomone-hub/genesis/issues/22)                    | moul                 | Questions about the connection between the AtomOne repository and Tendermint Atom One. Suggestions for a constitution and clarity on PHOTON design. |
| [<ins>#23: Issues/PRs Moderation</ins>](https://github.com/atomone-hub/genesis/issues/23)                            | moul                 | Challenge of selecting a responsible party for moderation. Proposal of an operations committee for moderation decisions.                            |
| [<ins>#24: Setup Weekly Report</ins>](https://github.com/atomone-hub/genesis/issues/24)                              | moul                 | Idea of initiating a weekly report. Use tools like ChatGPT to compile the list, with human intervention for fine-tuning.                      |

> | [<ins>#25: Should AiB Hire Dedicated Resources for Atomone?</ins>](https://github.com/atomone-hub/genesis/issues/25) | moul                 | Proposal to create a job for overseeing Atomone at AiB. Its role is to facilitate progress and act as a bridge between the community and AiB.                  |

Reviewer: ```suggestion
| [<ins>#25: Should AiB Hire Dedicated Resources for Atomone?</ins>](https://github.com/atomone-hub/genesis/issues/25) | moul                 | Proposal to create a job for overseeing AtomOne at AiB. Its role is to facilitate progress and act as a bridge between the community and AiB.                  |
```

- Proposal for a community-driven treasury oversight committee.

### Choosing the Genesis Software Target

[<ins>Issue #17</ins>](https://github.com/atomone-hub/genesis/issues/17) discusses the initial 
software choice for AtomOne.

#### Key points:

- Starting with Tendermint 2 (tm2) requires additional development.
- Starting with a Cosmos Hub 4 Fork allows for an early start but involves complex migration to tm2.

### Choose Snapshot Attributes

[<ins>Issue #18</ins>](https://github.com/atomone-hub/genesis/issues/18) focuses on selecting 
the appropriate block for a snapshot in AtomOne.

#### Key points:

- Debate on using specific blocks for the snapshot.
- Consideration of other proposals in snapshot attribute determination.

### Link with [Tendermint AtomOne Repository](https://github.com/tendermint/atom_one)

[<ins>Issue #22</ins>](https://github.com/atomone-hub/genesis/issues/22) seeks to clarify the 
relationship between AtomOne Hub's GitHub repository and the Tendermint repository.

#### Key points:

- Discussion on the relationship between AtomOne Hub's and Tendermint's repositories.
- Need for a constitution and clarity on PHOTON design.

> ### Should Atomone be an ICS Consumer for any Blockchain?

Reviewer: ```suggestion
### Should AtomOne be an ICS Consumer for any Blockchain?
```

security standards into proposal evaluations for AtomOne.

#### Key points

- Proposals to include verifiable security standards, potentially marked by a 'security coverage' badge.
- Risk, security, and financial score integration for robust proposal assessment.
- Community-led review and feedback mechanism for proposals, with support for specialized security groups.
- Emphasis on balanced decision-making, combining strict security measures with adaptability for urgent
  proposals.

---

## Branding and Public Relations

### Brand Identity and Twitter Handle (@_atomone)

[<ins>Issue #14</ins>](https://github.com/atomone-hub/genesis/issues/14) focuses on developing AtomOne's 
brand identity and Twitter handle (@_atomone).

#### Key points:

- Proposal for AtomOne’s brand identity and Twitter handle reservation.
- Move towards natural element motifs, focusing on the Hydrogen molecule.
- Significance of Hydrogen in symbolizing fundamental properties.
- Design proposals with community feedback and development suggestions.

### ATOM-ONE Secure Communications for Users

[<ins>Issue #21</ins>](https://github.com/atomone-hub/genesis/issues/21) highlights the need for secure 

> communication alternatives for ATOM-ONE users.

Reviewer: ```suggestion
communication alternatives for AtomOne users.
```

#### Key points:

- Need for end-to-end encrypted communication channels.
- An official, secure app or platform for governance communication is proposed.

---

## Stakeholder Relations

### Fix Cosmos Ecosystem Power Dynamics

[<ins>Issue #19</ins>](https://github.com/atomone-hub/genesis/issues/19) addresses the power dynamics 
within the Cosmos Ecosystem.

#### Key points:

- Need for a primary chain supporting all entities in Cosmos, reducing political complexity.
- Proposal for an inclusive and equitable ecosystem.

### Airdrops and Unstaking due to Emotional Distress Against the Halving

[<ins>Issue #33</ins>](https://github.com/atomone-hub/genesis/issues/33) focuses on supporting users 
who unstaked ATOM due to recent proposals.

#### Key points

- Support for creators and Atom1 supporters who left the network.
- Concerns about governance direction and proposals.
- Challenges and solutions for rewarding supporters.
- Rewarding contributors and creators within the AtomOne ecosystem.

### Official Cosmos ATOM-ONE NFT

[<ins>Issue #20</ins>](https://github.com/atomone-hub/genesis/issues/20) explores creating an official 
AtomOne NFT.

#### Key points:

> - NFT as recognition for early Atom-One project supporters.

Reviewer: ```suggestion
- NFT as recognition for early AtomOne project supporters.
```

list for AtomOne.

#### Key points:

- Setting up a list of interested contributors.
- Mechanism for organization and participation.
- Proposed method for member listing.
- Importance of transparency and public engagement.

---

## Resources

### Understanding AtomOne

[<ins>Issue #36</ins>](https://github.com/atomone-hub/genesis/issues/36) presents a metaphorical narrative 
to understand AtomOne.

#### Key points:

- AtomOne compared to essential water and its inflation to a water source.
- Authority and regulation by the Decentralists.
- Inflation and tokenized trust implications.
- Alternative analogies and sustaining inflation.

---

## Logistics

### Issues/PRs Moderation

[<ins>Issue #23</ins>](https://github.com/atomone-hub/genesis/issues/23) discusses effective moderation of 
issues and PRs in AtomOne's GitHub repository.

#### Key points:

- Debate on who should lead in moderating the GitHub repository.
- Proposed structures for moderation and the use of tools like ChatGPT.

### Setup Weekly Report

[<ins>Issue #24</ins>](https://github.com/atomone-hub/genesis/issues/24) proposes creating a weekly report 

> for the Atom-One project.

Reviewer: ```suggestion
for the AtomOne project.
```
